Package.swift

```diff
@@ -6,6 +6,7 @@ import PackageDescription
 let package = Package(
     name: "ContentScopeScripts",
     products: [
+        .executable(name: "build-node", targets: ["BuildNode"]),
```

Comment:
From what I've learned, this will never run by default. making it perfect for our use-case